lib/internal/Magento/Framework/View/Test/Unit/Layout/Argument/Interpreter/ObjectTest.php

```diff
@@ -22,19 +22,44 @@ class ObjectTest extends \PHPUnit\Framework\TestCase
     protected $_interpreter;
 
     /**
-     * @var DataObject
+     * @var \Magento\Framework\View\Layout\Argument\Interpreter\DataObject
      */
     protected $_model;
 
     protected function setUp()
     {
         $this->_objectManager = $this->createMock(\Magento\Framework\ObjectManagerInterface::class);
-        $this->_model = new DataObject($this->_objectManager, self::EXPECTED_CLASS);
+        $booleanUtils = $this->createMock(\Magento\Framework\Stdlib\BooleanUtils::class);
```

Comment:
Making `BooleanUtils` mock a local in this method would make it impossible to mock its method `toBoolean` in the test methods to check different scenarios. Please convert `$booleanUtils` to class variable and mock its method in the tests correspondingly